.github/workflows/test-integration.yml

```diff
@@ -0,0 +1,31 @@
+name: Test Integration
```

Comment:
Added this because overlay won't run on mac's so bumping the csi-tests out to be more integration sanity tests.